src/hotspot/share/opto/callnode.cpp

```diff
@@ -1463,11 +1463,20 @@ void SafePointNode::disconnect_from_root(PhaseIterGVN *igvn) {
 
 //==============  SafePointScalarObjectNode  ==============
 
-SafePointScalarObjectNode::SafePointScalarObjectNode(const TypeOopPtr* tp, Node* alloc, uint first_index, uint n_fields) :
+SafePointScalarObjectNode::SafePointScalarObjectNode(const TypeOopPtr* tp,
```

Comment:
is this a new commit on upstream that is missing in ms-patches/reduce-allocation-merges?

Reply:
Yes, there was a change recently made to the same lines that I changed while backporting RAM.

Reply:
Looks good to me no merge conflicts